docs/api.md

```diff
@@ -1202,6 +1208,12 @@ for the authenticated user.
   
   <!--end-response-body-get-accountsessions-isCurrentDevice-->
 
+* `isMemoryToken`: *boolean, required*
```

Comment:
>I wonder if a better abstraction here would involve us sending two separate timestamps 

Yeah we had that idea earlier, but I Think @philbooth wanted to keep the fields from redis and mysql consistent (to avoid confusion, etc)